samples/cuda_samples/CMakeLists.txt

```diff
@@ -132,10 +77,70 @@ add_cuda_sample(cuda-sobolqrng  4_Finance/SobolQRNG sobol.cpp  sobol_gold.cpp  s
 add_cuda_sample(cuda-reduction  6_Advanced/reduction reduction.cpp  reduction_kernel.cu)
 add_cuda_sample(cuda-fastwalsh  6_Advanced/fastWalshTransform fastWalshTransform.cu  fastWalshTransform_gold.cpp)
 
-# 6_Advanced/shfl_scan/shfl_integral_image.cuh:31:33: error: use of undeclared identifier 'warpSize'
-# 6_Advanced/shfl_scan/shfl_integral_image.cuh:83:17: error: no matching function for call to '__shfl_up'
-# error: no matching function for call to '__shfl_xor'
-#add_cuda_sample(cuda-shflscan  6_Advanced/shfl_scan shfl_scan.cu)
+############################
+
+if(NOT ENABLE_FAILING_TESTS)
+  ##### when run, sometimes fails sometimes passes TODO investigate
+  set_tests_properties("cuda-simpleCallback" PROPERTIES DISABLED 1 LABELS "")
+endif()
+
```

Comment:
This test does not fail - should be enabled.

Reply:
It has been failing sporadically before; now that i rebased the PR on master, with Level0 backend it still fails about 1 in 20 runs, with OpenCL backend about 1 in 5. I don't think it should be enabled until it *reliably* passes.

Reply:
I just spent ~10min running `cuda_simpleCallback` on L0 non-stop and it hasn't failed once. `Main` branch, `Debug` config.

Reply:
I have just tried with commit 96537f11ac56d2 and it still fails as i described, after about 10-30 runs. Hardware is Intel Skylake i5-6400, intel driver is 22.31.23852. Compiled as Debug build with "-Og" optimization level.

Reply:
@franz could you please try with `intel_compute_runtime/release/22.22.23355` ? If that passes it would imply a regression in the runtime because I tried several so far and they all pass but are also older than the one you use.

Reply:
@pvelesko  tested with 22.22.23355, does not seem to make any difference.

Reply:
I can reproduce it on Arcticus but not Iris. Let's go ahead and mark this one as flaky and circle back to it after the `hipStreamPerThread` merge

Reply:
I can also reproduce this on Iris using OpenCL backend.

Reply:
OK, lets keep this test disabled for now

Reply:
Is there an alternative mechanism for disabling tests? This would require modifying the CMakeLists.txt for HIP-COMMON